metomi/rose/apps/fcm_make.py

```diff
@@ -173,6 +176,7 @@ def _run_orig(self, app_runner, conf_tree, opts, args, uuid, task,
         # Determine the name of the continuation task
         task_name_cont = task.task_name.replace(
             orig_cont_map[ORIG], orig_cont_map[CONT])
+        # TODO: get_task_auth currently does nothing
```

Comment:
Is this TODO important before beta? Just wondering if it was intended to be in this pr?

Reply:
Kinda but we can limp on without it for a short time, here's the issue - https://github.com/metomi/rose/issues/2445